src/main/webapp/app/entities/theory/theory.component.html

```diff
@@ -1,67 +1,47 @@
-            <th jhiSortBy="content"><span jhiTranslate="learnportalApp.theory.content">Content</span> <fa-icon [icon]="'sort'"></fa-icon></th>
-            <th jhiSortBy="category.fullName"><span jhiTranslate="learnportalApp.theory.category">Category</span> <fa-icon [icon]="'sort'"></fa-icon></th>
-            <th></th>
-            </tr>
-            </thead>
-            <tbody infinite-scroll (scrolled)="loadPage(page + 1)" [infiniteScrollDisabled]="page >= links['last']" [infiniteScrollDistance]="0">
-            <tr *ngFor="let theory of theories ;trackBy: trackId">
-                <td><a [routerLink]="['/theory', theory.id, 'view' ]">{{theory.id}}</a></td>
-                <td>{{theory.title}}</td>
-                <td>{{theory.description}}</td>
-                <td>{{theory.content}}</td>
-                <td>
-                    <div *ngIf="theory.category">
-                        <a [routerLink]="['/category', theory.category?.id, 'view' ]" >{{theory.category?.fullName}}</a>
-                    </div>
-                </td>
-                <td class="text-right">
-                    <div class="btn-group">
-                        <button type="submit"
-                                [routerLink]="['/theory', theory.id, 'view' ]"
-                                class="btn btn-info btn-sm">
-                            <fa-icon [icon]="'eye'"></fa-icon>
-                            <span class="d-none d-md-inline" jhiTranslate="entity.action.view">View</span>
-                        </button>
-                        <button type="submit"
-                                [routerLink]="['/theory', theory.id, 'edit']"
-                                class="btn btn-primary btn-sm">
-                            <fa-icon [icon]="'pencil-alt'"></fa-icon>
-                            <span class="d-none d-md-inline" jhiTranslate="entity.action.edit">Edit</span>
-                        </button>
-                        <button type="submit"
-                                [routerLink]="['/theory', { outlets: { popup: theory.id + '/delete'} }]"
-                                replaceUrl="true"
-                                queryParamsHandling="merge"
-                                class="btn btn-danger btn-sm">
-                            <fa-icon [icon]="'times'"></fa-icon>
-                            <span class="d-none d-md-inline" jhiTranslate="entity.action.delete">Delete</span>
-                        </button>
+
+    <div class = "container col-offset-1">
+        <div class="row">
+            <div class="input-group mb-3 col">
+                <input [(ngModel)]="searchValue" (ngModelChange)="onSearchChange()" class="form-control" placeholder="{{'entity.action.liveSearch'| translate}}" type="text"> </div>
+            <div class="input-group mb-3 col-3">
+                <select [(ngModel)]="categoryId" (ngModelChange)="onSearchChange()" style = "width: 100%;" class="custom-select" id="inputGroupSelect01">
```

Comment:
Delete all hardcoded styles from HTML. They needs to be placed only to SCSS files created for every angular component!